docs/technical-overview/feature-flags.md

```diff
@@ -4,19 +4,14 @@ sidebar_position: 6
 
 # Feature flags
 
-:::important
-
-We’re currently making rapid changes to the product so our docs may be out of date. If you need help, please email [yo@forem.com](mailto:yo@forem.com).
-
-:::
-
 We sometimes employ feature flags to develop features that aren't fully formed
 at the moment of deployment. They can also be employed to roll out features in
 production incrementally.
 
-Feature flags are meant to be temporary and part of a rollout plan resulting in
+Feature flags are meant to be **temporary** and part of a rollout plan resulting in
 their removal.
 
+Since we do not expose feature flags to Forem Creators, we have built in the concept of an extension which ties in closely with Feature Flags.
```

Comment:
I did some searching on Forem.dev -- it looks like Extensions can be enabled at `/admin/advanced/extensions`. I didn't take that much time to look around on Forem.dev / developers.forem.com, but wondering if we have a page that talks a bit more about what Extensions are.

Also, curious (I don't think this needs to go into the docs page), do we have any guidelines on when to create an Extension when first starting the development of a feature? Do we see Extensions as a place for future external add ons from the community (like plugins in WordPress, modules in Drupal)?

Reply:
> but wondering if we have a page that talks a bit more about what Extensions are.

We don't have one at the moment, I think we all have a very basic understanding that if a feature flag is for Forem Creators that we should expose it as extensions but we haven't fully fledged out the "Extensions" concept especially since our focus is dev.to at the moment. 

> Also, curious (I don't think this needs to go into the docs page), do we have any guidelines on when to create an Extension when first starting the development of a feature? Do we see Extensions as a place for future external add ons from the community (like plugins in WordPress, modules in Drupal)?

I think this ties into the above, I think the guideline at the moment is that Feature Flags should be for developers to use and if they are going to be used by Forem Creators than that needs to be turned into an extension. Extensions can be turned on/off based on Creator's need. For example, Listings or Connect (although eventually we removed Connect)


> Do we see Extensions as a place for future external add ons from the community (like plugins in WordPress, modules in Drupal)?

We started talking about this a year ago but then put the brakes on it and hence we do not really have any concreate plans with it as the moment.